src/pages/chat/components/add-user-card/add-user.ts

```diff
@@ -0,0 +1,72 @@
+import Block from '../../../../utils/Block';
+import { Input } from '../../../../components/input';
+import { Button } from '../../../../components/button';
+import addUserTmpl from './add-user.tmpl';
+import ChatController from '../../../../controllers/ChatController';
+import store from '../../../../utils/Store';
+
+type AddUserProps = {
+    inputId?: Input;
+    addButton?: Button;
+    cancelButton?: Button;
+    events?: {
+        submit: (e: Event) => void;
+    }
+}
+
+class AddUser extends Block<AddUserProps> {
+    constructor(props: AddUserProps) {
+        super({
+            ...props,
+            events: {
+                submit: (e: Event) => this.handlerSubmit(e)
+            },
+            inputId: new Input({
+                type: 'text',
+                title: 'Пользователь',
+                name: 'user'
+            }),
+            addButton: new Button({
+                type: 'submit',
+                text: 'Добавить',
+                className: 'btn-primary btn-small'
```

Comment:
Последние строки в таких случаях лучше тоже заканчивать на `,`, потому что это положительно сказывается на "диффах" в системах контроля версий
```suggestion
                type: 'text',
                title: 'Пользователь',
                name: 'user',
            }),
            addButton: new Button({
                type: 'submit',
                text: 'Добавить',
                className: 'btn-primary btn-small',
```